aggregations-and-analytics/10/10_windowing_tvf.md

```diff
@@ -0,0 +1,64 @@
+# 10 Windowing table-valued functions
+
+![Twitter Badge](https://img.shields.io/badge/Flink%20Version-1.13%2B-lightgrey)
+
+> :bulb: This example will show how to apply computations on data in a temporal window using Windows table-valued functions `TUMBLE`, `HOP` and `CUMULATE`.
+
+The source table (`server_logs`) is backed by the [`faker` connector](https://flink-packages.org/packages/flink-faker), which continuously generates rows in memory based on Java Faker expressions.
+
+#### What are Windowing Table-valued functions (TVFs)?
+
+Windows split the stream into “buckets” of finite size, over which we can apply computations.
```

Comment:
Windowing TVFs don't support session windows yet, is that correct?

Reply:
Session windows are indeed not yet supported. I'm considering to not name this section 'Windowing TVF', because most of the interesting analytical calculations will be done via TVFs in the end.

I don't think we should yet replace the existing Group Window Functions, because a lot of end users are not yet on Flink 1.13+. I prefer to keep the existing locations (to avoid that we break any external links) but make a note that these recipes are outdated and we recommend to follow an updated recipe (with a link to the successor).